File: src/main/java/Main/ItemClasses/Potion.java
package Main.ItemClasses;

public class Potion extends Consumable {
    public Potion(String name, String description, int weight, int valueInGold, int amount) {
        super(name, description, weight, valueInGold, amount);
        this.iconName = "Potion - Rainbow.png";
    }

    public Potion() {
        super();
        this.iconName = "Potion - Rainbow.png";
    }
}

File: src/main/kotlin/main/ItemClasses/Potion.kt
package main.ItemClasses

import kotlinx.serialization.Serializable

@Serializable
class Potion() : Consumable() {
    init {
        standardIconName = "Potion - Rainbow.png"
    }

    constructor(name: String, description: String, weight: Int, valueInGold: Int, amount: Int) : this() {
        this.name = name
        this.description = description
        this.weight = weight
        this.valueInGold = valueInGold
        this.amount = amount
    }
}
